part_1/1.4_conditionals.md

```diff
@@ -57,10 +67,222 @@ When you run your code, if the age condition is met you should see the sentence
+explore using tests to define Expected Behavior.*
+
+## Else If Statements
+
+Sometimes when programming we are given problems where we have to define multiple patterns of behavior based on met conditions. We going to 
+continue using the *movie theater* problem to explore this. 
+
+**Problem:**
+
+The local movie theatre has noticed that some small children do not enjoy watching *The Princess Bride*. The movie theatre wants to offer tickets
+to *The Little Mermaid* for patrons who are 7 years old and younger. 
+
+How do we do this?
+
+Using only the tools we have so far the code might look something like this. 
+
+```go 
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	age := 10
+	if age >= 17 {
+		fmt.Println("You are allowed to buy a ticket")
+	} else {
+		if age >= 7 {
+			fmt.Println("Would you like to watch the Little Mermaid")
+		} else {
+			fmt.Println("Why don't you try the Princess Bride instead")
+		}
+	}
+
+}
+```
+
+This works, but doesn't this look kinda gross? This code is hard to read and hard to know what the expected behavior is supposed to be. When code is hard to read the first thing we should ask ourselves is if there is better way to write this? 
+
+In Go, and most programming language, there is a construct called [`Else If`](https://gobyexample.com/if-else) that allows us to write this is a more readable way. `Else If` allows us to add additional condition checks to an `If Els` statement. For example: 
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	age := 6
+	if age >= 17 {
+		fmt.Println("You are allowed to buy a ticket")
+	} else if age <= 7 {
+		fmt.Println("Would you like to watch the Little Mermaid")
+	} else {
+		fmt.Println("Why don't you try the Princess Bride instead")
+	}
+}
+```
+
+That was so much easier. We can add as many `Else If` conditions as needed. 
 
 ## Switch Statements
 
+Personally, I do not use `Else If` statements when programming is Go. I rarely use `Else` statements. In Go there is another control flow construct called a [`Switch Statement`](https://gobyexample.com/switch). The `Switch` statement like `If Else` uses conditions to define a programs behavior. Let's use the previous example and apply it to our switch statement. 
+
+We ended with this chunk of code inside our if statement.
+
+```go
+	age := 6
+	if age >= 17 {
+		fmt.Println("You are allowed to buy a ticket")
+	} else if age <= 7 {
+		fmt.Println("Would you like to watch the Little Mermaid")
+	} else {
+		fmt.Println("Why don't you try the Princess Bride instead")
+	}
+```
+
+All of the logic we have in this code depends  on the value of the `age` variable. The syntax of the `Switch` statement highlights the dependency on a certain variable and uses the syntax of a `case` define the conditions that guide the code's behavior. There is also a case called `default` that is to run when none of the conditions are met. Here is what the code would look like as a `Switch` statement:
```

Comment:
I would recommend writing `Switch` as the lowercase `switch`, to match what gets written in go.